sources/api/migration/migrations/v1.0.6/admin-container-v0-6-0/src/main.rs

```diff
@@ -0,0 +1,29 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::common_migrations::ReplaceTemplateMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+const OLD_ADMIN_CTR_TEMPLATE: &str =
+    "{{ ecr-prefix settings.aws.region }}.amazonaws.com/bottlerocket-admin:v0.5.2";
+const NEW_ADMIN_CTR_TEMPLATE: &str =
+    "{{ ecr-prefix settings.aws.region }}.amazonaws.com/bottlerocket-admin:v0.6.0";
```

Comment:
The real values don't have the domain name (`ecr-prefix` takes care of that for you) so this wouldn't match.